docs/howto/operate/setup-grafana.md

```diff
@@ -0,0 +1,83 @@
+# Setting up Grafana Dashboards for a cluster
+
+This guide will walk through the steps required to setup a suite of Grafana dashboards for a cluster.
+
+## Before you start
+
+Make sure you are authenticated against the correct cloud platform and have selected the correct project to deploy to.
+
+For example, for Google Cloud:
+
+```bash
+gcloud auth application-default login
+gcloud config set project PROJECT_ID
+```
+
+## Deploy the `support` chart
+
+The `support` chart is a helm chart maintained by the 2i2c Engineers that consists of common tools used to support JupyterHub deployments in the cloud.
+These tools are [`ingress-nginx`](https://kubernetes.github.io/ingress-nginx/), for controlling ingresses and load balancing; [`cert-manager`](https://cert-manager.io/docs/), for automatically provisioning TLS certificates from [Let's Encrypt](https://letsencrypt.org/); [Prometheus](https://prometheus.io/), for scraping and storing metrics from the cluster and hub; and [Grafana](https://grafana.com/), for visualising the metrics retreived by Prometheus.
+
+### Edit your `*.cluster.yaml` file
+
+Add the following config as a top-level key to your `*.cluster.yaml` file under `/config/hubs` in `pilot-hubs`.
+`GRAFANA_URL` should follow the pattern `grafana.<cluster_name>.2i2c.cloud`.
+
+```yaml
+support:
+  config:
+    grafana:
+      ingress:
+        hosts:
+          - GRAFANA_URL
+        tls:
+          - secretName: grafana-tls
+            hosts:
+              - GRAFANA_URL
+```
+
+### Deploy the `support` chart via the `deployer`
+
+If you don't have the `cert-manager` chart added to your helm setup already, you can add it and update it like so:
+
+```bash
+helm repo add jetstack https://charts.jetstack.io
```

Comment:
Let's just automate this with `deploy-support` instead? We don't nee to add the other repos (for nginx-ingress, prometheus, etc) because helm does it for us from the `Chart.yaml`. We'll just have to recreate a tiny bit of that.

Reply:
I had originally done this [here](https://github.com/2i2c-org/pilot-hubs/pull/548/commits/a2ddda1243d58cdc9d25f4af4205f108359a88c2), but wasn't certain that was the right choice and so migrated it into the action.yml file instead